templates/taxi/car_list.html

```diff
@@ -7,6 +8,11 @@ <h1>
       Create
     </a>
   </h1>
+  <form method="get" class="form-inline">
+    {{ search_form|crispy }}
+    <input class="btn btn-secondary" type="submit" value="search">
+  </form>
+  <br>
```

Comment:
will pagination work correctly with ur search ?

Reply:
added templatetag, forgot about it. ty!